label_studio/io_storages/aperturedb/form_layout.yml

```diff
@@ -0,0 +1,59 @@
+# 2x3 grid
+adb_host_config: &adb_host_config
+  - type: text
+    name: title
+    label: Storage Title
+  - type: text
+    name: hostname
+    label: Hostname
+    required: true
+  - type: number
+    name: port
+    label: Port
+  - type: text
+    name: username
+    label: User Name
+    required: true
+  - type: password
+    name: password
+    label: Password
+    autoComplete: "off"
+    skipAutofill: true
+    allowEmpty: true
+    protectedValue: true
+  - type: password
+    name: token
+    label: Token
+    autoComplete: "off"
+    skipAutofill: true
+    allowEmpty: true
+    protectedValue: true
+
+ImportStorage:
+  - columnCount: 3
+    fields: *adb_host_config
```

Comment:
very cool that you can reference other places. Do you know if this is standard yaml or an extension?

Reply:
According to [Wikipedia](https://en.wikipedia.org/wiki/YAML):
> Repeated nodes are initially denoted by an [ampersand](https://en.wikipedia.org/wiki/Ampersand) (&) and thereafter referenced with an [asterisk](https://en.wikipedia.org/wiki/Asterisk) (*).